wagtail/snippets/wagtail_hooks.py

```diff
@@ -66,17 +63,6 @@ def register_snippet_listing_buttons(snippet, user, next_url=None):
     viewset = model.snippet_viewset
     permission_policy = viewset.permission_policy
 
-    if user_can_edit_snippet_type(user, model):
-        yield SnippetListingButton(
-            _("Edit"),
-            reverse(
-                viewset.get_url_name("edit"),
-                args=[quote(snippet.pk)],
-            ),
-            attrs={"aria-label": _("Edit '%(title)s'") % {"title": str(snippet)}},
-            priority=10,
-        )
-
     if viewset.inspect_view_enabled and permission_policy.user_has_any_permission(
```

Comment:
After #10931, we'll be able to remove this file completely, so that all default buttons are defined in the generic `IndexView`.